src/layers/Tooltip/Tooltip.tsx

```diff
@@ -191,6 +206,25 @@ export const Tooltip: FC<Partial<TooltipProps>> = ({
 
         return (
           <motion.div
+            onMouseOver={() => {
+              // keep popover open when interacting with it
+              if (
+                isPopover &&
+                (trigger === 'hover' ||
+                  (Array.isArray(trigger) && trigger.includes('hover')))
```

Comment:
minor optimization - you could make this a variable like `isPopoverInteraction` and also reuse it for `onMouseLeave`